go_logging_server/logging/logging.go

```
@@ -39,37 +43,95 @@ const (
+	}
+
+	//Find index for service
+	serviceIndex = utils.IndexOf(logIn.Service, services)
+	if serviceIndex == -1 {
+		return fmt.Errorf("failed to write to log could not find file for service %s", logIn.Service)
+	}
+
+	//Get file
+	file = files[serviceIndex].file
+
+	file.WriteString(fmt.Sprintln("\n--------------\n" + level + "\n[" + logIn.Service + "]\n" + logIn.TimeStamp + "\n" + logIn.Message))
```

golangci-lint (1.64.8)

Error return value of `file.WriteString` is not checked

(errcheck)